griptape/drivers/image_query/anthropic_image_query_driver.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional
+from attr import define, field, Factory
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseImageQueryDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from anthropic import Anthropic
+
+
+@define
+class AnthropicImageQueryDriver(BaseImageQueryDriver):
+    """
+    Attributes:
+        api_key: Anthropic API key.
+        model: Anthropic model name.
+        client: Custom `Anthropic` client.
+        max_output_tokens: Max output tokens to return.
+    """
+
+    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    model: str = field(default="claude-3-sonnet-20240229", kw_only=True, metadata={"serializable": True})
```

Comment:
Remove default model value.

Reply:
Can do, but I was curious about this. I saw we set defaults for OpenAI drivers, but not others. Why is that?

Reply:
Ah I didn't realize we set in openai. That probably shouldn't be set their either. But for sake of consistency we can keep it.

@andrewfrench thoughts?

Reply:
I had already removed the anthropic default. I can remove the openai default as well in the name of consistency.